.gitignore

```diff
@@ -1,3 +1,4 @@
 .idea/*
 *.iml
 target/*
+ConnectionUtil
```

Comment:
You add `ConnectionUtil` to .gitignore, but I still see this class. Check this please :)